src/tests/build.proj

```diff
@@ -9,6 +9,7 @@
     <MSBuildEnableAllPropertyFunctions>1</MSBuildEnableAllPropertyFunctions>
     <Language>C#</Language>
     <RuntimeIdentifier>$(OutputRid)</RuntimeIdentifier>
+    <RestoreAdditionalProjectSources Condition="$(TargetArchitecture) == 'loongarch64'">$(RestoreAdditionalProjectSources);$(ArtifactsPackagesDir)</RestoreAdditionalProjectSources>
```

Comment:
This makes the build use packages that were just build instead of the one from the feed. Can we unify this with the mechanism that source build uses to achieve the same?

cc @MichaelSimons

Reply:
Why is this needed? Which live built packages do you need to restore and reference here?

Reply:
> This makes the build use packages that were just build instead of the one from the feed. Can we unify this with the mechanism that source build uses to achieve the same?

Source-build does not have a special mechanism to do this within a repo.  The source-build mechanism is used to pickup packages that were built from a different repo within the source build.

Reply:
Technically this line not needed, and only similar line in the file is here. But I assume concerns applied to line below.

Reply:
@ViktorHofer isn't there some other way of forcing tests to use the live built packages?

Reply:
@kant2002 I'm curious does the restore error only happen for the test_dependencies.fsproj you mentioned in https://github.com/dotnet/runtime/issues/78854?

Reply:
I do not actually know, since I do not fully understand how test infra setup. So I was worrying that I may break something further down the road. I can play according to your directions if you have some ideas to test